app-android/src/main/java/com/ustadmobile/port/android/view/SiteDetailFragment.kt

```diff
@@ -183,4 +199,49 @@ class SiteDetailFragment: UstadDetailFragment<Site>(), SiteDetailView, Workspace
 
     }
 
-}
\ No newline at end of file
+}
+
+@Composable
+fun SiteDetailScreen(
+    uiState: SiteDetailUiState,
+    onClickLang: (SiteTermsWithLanguage) -> Unit = {}
+){
+
+    Column {
+        UstadDetailField(
+            valueText = uiState.site?.siteName ?: "",
+            labelText = stringResource(R.string.name),
+            imageId = R.drawable.ic_account_balance_black_24dp
+        )
+        UstadDetailField(valueText = stringResource(if(uiState.site?.guestLogin == true){R.string.yes} else {R.string.no}), labelText = "Guest login enabled", imageId = R.drawable.ic_document_preview)
```

Comment:
These lines are too long. The gray line on the right hand side of the IDE indicates the maximum line length that should be used. E.g. should be split into three lines:

```
UstadDetailField(
            valueText = stringResource(if(uiState.site?.guestLogin == true){R.string.yes} else {R.string.no}), 
            labelText = stringResource(R.string.guest_login_enabled), 
            imageId = R.drawable.ic_document_preview
        )
```

Please make sure to always use stringResource, not just the hardcoded string in English (e.g. labelText = stringResource(..)). Otherwise it will not be translated when the app is used in another language.